### ISSUE TITLE
High temperature on AL-BOXⅡ

### ISSUE THREAD

**Issue by fodger** [goldshellminer/firmware#105]
Hello,
Temperature is about 80° on AL-BOXⅡ, too too too high !
While at starting we hear that fans can work harder.

You must give a firmware that allows to control fan rate please !

Thanks a lot.

**Comment by Ossbad** [comment]
Yes, we need more control fans for coping down, also after the last update the profit became less.

**Comment by Sykes1** [comment]
And the original firmware is not archived. HELP! Need an update or the old file.

**Comment by The-1-C** [comment]
Has anyone got an update for this?

**Comment by Sykes1** [comment]
I emailed Goldshell and the sent me the original firmware. I recommend you reach out to them.

**Comment by The-1-C** [comment]
> I emailed Goldshell and the sent me the original firmware. I recommend you reach out to them.

I just created a support ticket, did the original firmware work for you?
